vuepress/docs/next/tutorials/consume/multitenancy-tutorial.md

```diff
@@ -21,97 +21,55 @@ See [Multitenancy on Entando](../../docs/consume/multitenancy.md), for details o
-2. Follow this tutorial to [Create a Backup of the Keycloak Realm](../devops/backing-restoring-keycloak.md). 
+Import the realm for Keycloak from the primary tenant and reconfigure it for the secondary tenant.
+1. Log in to your [Keycloak admin console](../../docs/consume/identity-management.md#authorization), typically located at `http://YOUR-APPNAME.YOUR-HOSTNAME/auth`
+2. Follow this tutorial to [Create a Backup of the Keycloak Realm](../devops/backing-restoring-keycloak.md) 
 3. Edit the generated JSON file of the realm with these updates:
-     * Remove the many `id` attributes in the file.  The YOUR-TENANT1-CODE will become the prefix of the domain host name or URL. 
-     * Replace the properties `realm` & `displayName` with YOUR-TENANT1-CODE.
-     * Add the YOUR-TENANT1-CODE + "." as a prefix to the following fields:
-`redirectUris` and `webOrigins` under the clientId for `entando-web`, `quickstart` and `quickstart-de` sections.  
+     * Remove every `id` attribute in the file.  
+     * Replace the properties `realm` & `displayName` with YOUR-TENANT1-ID.
+     * Add `YOUR-TENANT1-ID` + "." as a prefix to the following fields:
+`redirectUris` and `webOrigins` under `clientId` with `entando-web`, `YOUR-APP-NAME` and `YOUR-APP-NAME-de`.  
 
-For example: 
+For example, if `YOUR-APP-NAME` is quickstart, and `YOUR-TENANT1-ID` is 2ndtenant1:
 ```
-"redirectUris": [
-    	"http://YOUR-TENANT1-CODE.YOUR-HOSTNAME/entando-de-app/*"
-  	],
+ "clientId" : "entando-web",
+ "redirectUris" : [ "https://2ndtenant1.quickstart.k8s-entando.org/*" ],
+ "webOrigins" : [ "https://2ndtenant1.quickstart.k8s-entando.org" ],
 ```
-4. Save the edited file as `realm-YOUR-TENANT1-CODE.json`.
+4. Save the edited file as `realm-YOUR-TENANT1-ID.json`.
 
 5. In the Keycloak admin console, click `Entando` at the top of the left navigation bar. Click `Add Realm` from the drop-down menu. Select your tenant's JSON file or enter the name. The `Enabled` button should be "On" to access a new realm.
-6. In the new realm, go to `Clients` and choose `Edit` under `Actions` for `quickstart` Client ID. Under the `Credentials` tab, regenerate the `Secret`. 
-7. Repeat step 6 for `quickstart-de` Client ID. 
-8. Create an `admin` user to manage the users and roles for this realm. Go to `Manage Users` from the left navigation options and [assign `realm-management` Client Role](../../docs/consume/identity-management.md#authorization) to the admin user.
+6. In the new realm, go to `Clients` and choose `Edit` under `Actions` for your application name under the list of Client IDs. Under the `Credentials` tab, regenerate the `Secret`. 
+7. Repeat step 6 for `YOUR-APP-NAME-de` Client ID. 
+8. Create an `admin` user to manage the users and roles for this realm. Go to `Manage Users` from the left navigation options and [choose `realm-management` Client Role and select `manage-realm` role](../../docs/consume/identity-management.md#authorization) for the admin user. 
```

Comment:
@nshaw @avdev4j  unsure how to confirm if this role assignment for admin is correct. I was told realm-management